### ISSUE TITLE
FR: Copy frame label (if present) instead of generating a random identifier

### ISSUE THREAD

**Issue by alkuzman** [zsviczian/obsidian-excalidraw-plugin#2029]
### Is your feature request related to a problem? Please describe.

When copying a frame from one drawing, the link always results in a randomly generated identifier instead of taking the label if present. This can easily result in broken links. So far, I am changing the links manually with the link editor.

### Describe the solution you'd like

Always copy the link with the frame label (if present) in the clipboard.

### Real-life usage scenarios

1. Prevents broken links in the future.
2. Renaming can also be handled, by updating all the references, but that is another feature.

### Describe alternatives you've considered

1. Manually changing the links.
2. Leaving the links as they are, but this does not make sense to me, since I usually have the same frame labels in most of the drawings, I can easily reference them.

**Comment by zsviczian** [comment]
yeah - I was thinking about this, however, the frame name may change, the frame ID stays constant.
but I can see indeed this as a future feature where you can choose to copy the link with the frame name or the frame id.
Note it is not a generated random ID, it is the frame id in the drawing.
